package/MDAnalysis/analysis/waterdynamics.py

```diff
@@ -508,11 +508,22 @@ class HydrogenBondLifetimes(object):
     .. versionchanged:: 1.0.0
        The ``nproc`` keyword was removed as it linked to a portion of code that
        may have failed in some cases.
+    .. deprecated:: 2.0.0
+       Instead, please use
```

Comment:
The deprecation will have to be in a 1.0.x
```suggestion
    .. deprecated:: 1.0.1
       ``waterdynamics.HydrogenBondLifetimes`` is deprecated and will be removed in 2.0.0.
       Instead, please use (available in 2.0.0)
```